data/tutorials/getting-started/gs_02_fst_program.md

```diff
@@ -0,0 +1,356 @@
+
+In other words, we do breadth-first learning instead of depth-first learning. In the previous tutorial most commands were entered in Utop, in this tutorial, the majority of commands should be entered into a terminal. Code examples starting with a dollar sign `$` are intended to be entered in the terminal, while lines starting with a hash sign `#` are intended to be entered in Utop.
+
+## Prerequisites and Goals
+
+This tutorial is the last part of a two-part series. Please ensure you have completed [A Tour of OCaml](/docs/a-tour-of-ocaml) before proceeding onto your first OCaml project. You will need to have [installed OCaml](/install).
+
+Once you've completed this tutorial, you should be able to:
+- Create an OCaml project from scratch, using Dune or manually
+- Trigger prject compilation and execution using Dune
+- Delete files in an automatically created project without breaking everything
+- Split code in several files, use imported definitions
+- Make a definition private
+- Download, install and use a package from the open source repository
+
+How to work on several OCaml projects simultaneously is out of the scope of this tutorial. Currently (summer 2023), this requires using Opam local [_switches_](https://opam.ocaml.org/doc/man/opam-switch.html). This allows handling different sets of dependencies per project. Check the Best Practices document on [Dependencies](https://ocaml.org/docs/managing-dependencies) addressing that matter for detailed instructions. This document was writen and tested using a global switch, which is created by default when installing Opam and can be ignored in the beginning.
+
+## Compiling OCaml Programs
+
+By default, OCaml comes with two compilers, one translating sources into native binaries and another turning sources into a bytecode format. OCaml also comes with an interpreter for that bytecode format. Other compilers exist, for instance, [js_of_ocaml](https://ocsigen.org/js_of_ocaml) generates JavaScript. The toplevel uses the bytecode compiler; expressions are read, type-checked, compiled into bytecode and executed. The previous tutorial was interactive only, the toplevel was used. This tutorial gives a glimpse at batch processing only using the native compiler.
+
+We start by seting up a traditional “Hello World!” project using Dune, OCaml’s build system. Make sure to have installed version 3.7 or later. The following creates it under the name `hello`:
+
+```shell
+$ dune init proj hello
+Entering directory '/home/cuihtlauac/caml/ocaml.org/hello'
+Success: initialized project component named hello
+```
+
+Note: Throuout this tutorial, outputs generated Dune might vary slightly because of the Dune version installed. This tutorial shows the output for Dune 3.7. If you'd like to get the most recent version of Dune, run `opam update; opam upgrade dune` in a terminal.
```

Comment:
```suggestion
Note: Throughout this tutorial, outputs generated Dune might vary slightly because of the Dune version installed. This tutorial shows the output for Dune 3.7. If you'd like to get the most recent version of Dune, run `opam update; opam upgrade dune` in a terminal.
```